module/VuFind/src/VuFind/Db/Service/TagService.php

```diff
@@ -292,9 +292,9 @@ public function getListsForTag(
     /**
      * Unlink rows for the specified resource.
      *
-     * @param string|array|null $resource ID (or array of IDs) of resource(s) to
+     * @param mixed             $resource ID (or array of IDs) of resource(s) to
      * unlink (null for ALL matching resources)
-     * @param string|User       $user     ID of user removing links
+     * @param string|User|int   $user     ID of user removing links
```

Comment:
Maybe this should be "ID or entity representing user" to cover the multiple input options.